initial commit

gattytto opened the pull request:
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

NOTE: branch of flutter SDK is master because some stuff is only available in this branch (and the dev which still doesn't have milestone for required [fix](https://github.com/flutter/flutter/pull/62115)).

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
initial commit

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
fix builder ref name in gh action

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

ericwill commented on VERSION:
```diff
@@ -0,0 +1 @@
+master
```
This needs to be a plugin version.

gattytto committed:
set fixed version

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
set a fixed 1.22.0-12.1.pre version for versioning the sidecar
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
set fixed version

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
Set a fixed version tag 1.22.0-12.1.pre for versioning the sidecar
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
freeze

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
set to a specific state
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
fix

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
fix typo
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
static release 

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
go back to previous config around static release.
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
typo

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
more typos
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto committed:
current version used

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
exact version for a working release link
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

ericwill commented on VERSION:
```diff
@@ -0,0 +1 @@
+1.22.0-12.0
```
Is this the same version as what the plugin will be in the registry? The idea is that we want to keep the version field here and the plugin version the same.

gattytto replied:
it's the sdk version actually

benoitf replied:
@ericwill why the framework image needs to be the same than the extension version ?

if we have two VS Code extensions using flutter, they could reuse the same image

gattytto committed:
use plugin version

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
use extension versioning
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

gattytto commented:
@ericwill it's now the same version as extension.

gattytto committed:
bump to dev version

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
extension stable is 3.13 but dev with changes goes up to 3.15.0-dev (plus the changes for che).
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>
